frontend/src/components/Footer.vue

```diff
@@ -29,6 +30,9 @@ export default {};
   .footer .v-btn {
     color: #fff;
     text-decoration: none;
+    text-transform: capitalize;
+    font-size: 80%;
+    font-weight: normal;
```

Comment:
Normal is the default font-weight, is this an override?

Reply:
yeah this does seem unnecessary but i believe v-btn has a default font weight of 500. that's what the browser inspector is showing me. i think its in the base vuetify. 500 is a bit heavy if i am trying to match the style in ticket i think